test/test_sparse.cc

```diff
@@ -428,7 +428,7 @@ class TestLSKGES : public ::testing::Test
         uint32_t ctr_A0 = 42;
         uint32_t seed_A0 = 42000;
         RandBLAS::dense::DenseDist DA0 = {.n_rows = m0, .n_cols = n0};
-        RandBLAS::dense::fill_buff(A0.data(), DA0, RandBLAS::base::RNGState{ctr_A0, seed_A0});
+        RandBLAS::dense::fill_buff(A0.data(), DA0, RandBLAS::base::RNGState{{{ctr_A0}}, {{seed_A0}}});
```

Comment:
To me, this looks much harder to read. Is it necessary based on the changes you made to RNGState? If so, what changes lead to this new initialization syntax?

Reply:
Agree. The double braces disambiguate what we intend to initialize here. this can be simplified, for instance by adding constructors to `RNGState` that take single `uint32_t`'s. In going with this admittedly confusing syntax I was opting to reduce the code in `RNGState`. I'll push a simplification in a follow up

Reply:
resolved in 6166587ca